doc/lua_api.md

```diff
@@ -5466,21 +5466,44 @@ Call these functions only at load time!
     * Returns an integer object handle uniquely identifying the registered
       ore on success.
     * The order of ore registrations determines the order of ore generation.
+* `minetest.override_ore(name, ore definition)`
+    * Override existing ore.
```

Comment:
Unclear whether it overrides only the specified fields (like `minetest.override_item`) or whether it is replaced entirely by the specified table.